src/include/cell.i

```diff
@@ -75,6 +95,11 @@ __cell_pack_value_validity(WT_SESSION_IMPL *session, uint8_t **pp, wt_timestamp_
             WT_IGNORE_RET(__wt_vpack_uint(pp, 0, start_txn));
             LF_SET(WT_CELL_TXN_START);
         }
+        if (durable_stop_ts != WT_TS_MAX) {
```

Comment:
How about storing the diff of `durable_stop_ts - stop_ts` instead of `durable_stop_ts - durable_stop_ts`. With this way, the number of bytes required may be less.

Reply:
Done.